### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by LONGLONGTOM**
Haolong Liu's Community Contribution

### CODE REVIEW

File: ml_interview_concept_mindmap.Rmd [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Explanation for the community contribution project
+Haolong Liu
```
Comment (philkim72) [inline ml_interview_concept_mindmap.Rmd:2]: The second line should be blank, followed by the author name(s).

File: ml_interview_concept_mindmap.Rmd [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## Explanation for the community contribution project
```
Comment (philkim72) [inline ml_interview_concept_mindmap.Rmd:1]: The first line should start with a single hashtag #, followed by a single whitespace, and then the title.

**Review by philkim72 — CHANGES_REQUESTED** [review]
Please make the two changes specified in the comments.  Also you seem to have two other pull requests.  I think those pull requests should be canceled, since it would be easier for you to make the necessary changes and combine them into this pull request.

**Comment by LONGLONGTOM** [comment]
Hi, I made the changes. Thank you so much

**Comment by philkim72** [comment]
Thank you for your contribution! :tada:

**Pull request merged**